src/Microsoft.Health.Fhir.Core/Configs/CoreFeatureConfiguration.cs

```diff
@@ -106,5 +106,12 @@ public class CoreFeatureConfiguration
         /// SearchParameter cache synchronized across instances. Default is 60 seconds if not specified.
         /// </summary>
         public int SearchParameterCacheRefreshIntervalSeconds { get; set; } = 60;
+
+        /// <summary>
+        /// Gets or sets the maximum initial delay in seconds for the SearchParameter cache background service timer.
+        /// This random delay (0 to this value) staggers timer startup across instances to prevent thundering herd.
```

Comment:
I have never heard the term 'thundering herd' used to describe an influx of traffic due to start up across instances. I like it :)